### PR TITLE
UnixPB: Only install ssl test packages on x64 for Centos/RHEL

### LINKED ISSUE #3224: Ansible request: Remove libnss3.so & libnssutil3.so
Temporarily remove the installation of libnss3.so and libnssutil3.so which are causing issues on non-x64 platforms for Centos & RHEL distributions.

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #3224

Remove problematic SSL test related packages from distributions that are encountering issues on ARM / PPC64LE on RHEL, CentOS & UBI 8.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)

### CODE REVIEW

**Comment by steelhead31** [comment]
VPC In progress: https://ci.adoptium.net/job/VagrantPlaybookCheck/1736/

**Review by sxa — APPROVED** [review]
Probably won't merge this while we're in a release cycle to avoid updating the build environments so there may be an opportunity to remediate the problem rather than backing out before this gets merged.

**Comment by steelhead31** [comment]
this latest commit moves required packages to x64 installations of Centos & RHEL only, Im currently unable to find an ARM64 equivalent for Centos & RHEL

**Review by sxa — APPROVED** [review]

**Review by karianna — APPROVED** [review]

**Pull request merged**
